stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         component: {{ .Values.artifactory.name }}
         release: {{ .Release.Name }}
       annotations:
+        checksum/binarystore: {{ include (print $.Template.BasePath "/artifactory-binarystore.yaml") . | sha256sum }}
```

Comment:
Note that a change to `binarystore.xml` will not be applied automatically, as the files in `/artifactory_extra_conf` are copied to `/var/opt/jfrog/artifactory/etc` only once, at first startup. See #181 .

Reply:
Yeah I wasn't sure what this checksum was used for in the HA chart, so I figured it was better to copy it across as well.

Reply:
If this is what we have in the `ha` chart, you can leave it here too.